common/reviews/api/heft-dev-cert-plugin.api.md

```diff
@@ -6,8 +6,10 @@
 
 import type { IHeftPlugin } from '@rushstack/heft';
 
-// @public (undocumented)
+// @internal (undocumented)
```

Comment:
These should be public.

Reply:
okay I'll change the comments in the index files for these packages